arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -0,0 +1,111 @@
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class QueryFeaturesArcadeExpressionViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+        }
+    }
+    
+    /// The arcade expression evaluation operation.
+    var evaluateOperation: AGSCancelable?
+    
+    /// Make and load a map.
+    func makeMap() -> AGSMap {
+        // Create a portal item with the portal and item ID.
+        let portalItem = AGSPortalItem(portal: .arcGISOnline(withLoginRequired: false), itemID: "14562fced3474190b52d315bc19127f6")
+        // Make a map with the portal item.
+        let map = AGSMap(item: portalItem)
+        // Load the map.
+        map.load() { _ in
+            // Set the visibility of all but the RDT Beats layer to false.
+            map.operationalLayers.forEach { layer in
+                let currentLayer = layer as? AGSLayer
+                if currentLayer?.name == "Crime in the last 60 days" || currentLayer?.name == "Police Stations" {
+                    currentLayer?.isVisible = false
+                }
+            }
+        }
+        return map
+    }
+    
+    /// Evaluate the arcade expression for the selected feature at the map point.
+    func evaluateArcadeInCallout(for feature: AGSArcGISFeature, at mapPoint: AGSPoint) {
+        // Instantiate a string containing the arcade expression.
+        let expressionValue = "var crimes = FeatureSetByName($map, 'Crime in the last 60 days');\n" + "return Count(Intersects($feature, crimes));"
+        // Create an arcade expression using the string.
+        let expression = AGSArcadeExpression(expression: expressionValue)
+        // Create an arcade evaluator with the arcade expression and an arcade profile.
+        let evaluator = AGSArcadeEvaluator(expression: expression, profile: .formCalculation)
+        guard let map = mapView.map else { return }
+        let profileVariables = ["$feature": feature, "$map": map]
+        // Get the arcade evaluation result given the previously set profile variables.
+        evaluateOperation = evaluator.evaluate(withProfileVariables: profileVariables) { [weak self] result, error in
+            guard let self = self else { return }
+            if let result = result, let crimeCount = result.cast(to: .string) as? String {
+                // Dismiss progress hud.
+                UIApplication.shared.hideProgressHUD()
```

Comment:
```suggestion
            // Dismiss progress hud.
            UIApplication.shared.hideProgressHUD()
            guard let self = self else { return }
            if let result = result, let crimeCount = result.cast(to: .string) as? String {
```
Dismiss the HUD regardless result or error, so it won't overlay upon the alert.